src/pyedb/dotnet/edb_core/nets.py

```diff
@@ -472,7 +472,7 @@ def _get_points_for_plot(self, my_net_points):
     @pyedb_function_handler()
     def get_plot_data(
         self,
-        nets,
+        nets=None,
```

Comment:
Could you update the description to:
- add `optional` to `nets` types (this should already have been performed)
- specify that, by default, all nets will be considered